octue/mixins/pathable.py

```diff
@@ -55,13 +63,17 @@ def _path_prefix(self):
 
     @property
     def absolute_path(self):
-        """The absolute path of this resource"""
+        """The absolute path of this resource."""
+        if self._path_is_in_google_cloud_storage:
+            return self.path
         return os.path.normpath(os.path.join(self._path_prefix, self._path))
 
     @property
     def relative_path(self):
-        """The path of this resource relative to its base path"""
-        return os.path.relpath(self.absolute_path, self._base_path)
+        """The path of this resource relative to its base path."""
+        if self._base_path is None:
+            return None
```

Comment:
I originally did that but changed it to what's there now because you can't have a relative path without a base path. Both work but this one seemed more generally applicable